app/assets/stylesheets/application.css

```diff
@@ -23,3 +23,33 @@
  .in_panel {
    list-style-type: none;
  }
+
+
+ .hvr-grow {
+   display: inline-block;
+   vertical-align: middle;
+   -webkit-transform: translateZ(0);
+   transform: translateZ(0);
+   box-shadow: 0 0 1px rgba(0, 0, 0, 0);
+   -webkit-backface-visibility: hidden;
+   backface-visibility: hidden;
+   -moz-osx-font-smoothing: grayscale;
+   -webkit-transition-duration: 0.3s;
+   transition-duration: 0.3s;
+   -webkit-transition-property: transform;
+   transition-property: transform;
+ }
+ .hvr-grow:hover, .hvr-grow:focus, .hvr-grow:active {
+   -webkit-transform: scale(1.1);
+   transform: scale(1.1);
+ }
+
+
+
+ 
+
+ .hvr-grow
+ {
+   height: 20px;
+   width: 15px;
+ }
```

Comment:
Can you fix the spacing issues with these styles and then factor them out into their own file? Like application.js, there probably shouldn't be code here aside from manifest loading instructions.